nginx/Dockerfile

```diff
@@ -1,24 +1,25 @@
 # syntax=docker/dockerfile:1
 
 ARG openssl_version="1.1.1w"
-ARG openssl_dir="/usr/local/ssl"
+ARG openssl_dir="/opt/openssl"
 ARG nginx_version="1.24.0"
 ARG nginx_dir="/opt/nginx"
 
 # * Use Ubuntu 18.04 as the build image as it's known-good for TLS 1.0/1.1
 FROM ubuntu:18.04 AS build
 
 # * Install build dependencies
-RUN apt-get update
-RUN apt-get install -y \
+RUN apt-get update \
+	&& apt-get install -y \
```

Comment:
Why was this changed?

Reply:
According to https://docs.docker.com/develop/develop-images/instructions/#apt-get running them on separate steps can cause caching issues.

Reply:
I was not aware of this 👍 thank you, I did it like that for style reasons but if there's an issue with that then I'm fine changing it